plotly/tests/test_core/test_spectacle_presentation/test_spectacle_presentation.py

```diff
@@ -0,0 +1,207 @@
+                                                    'style': {'color': '#000016',
+                                                              'fontFamily': 'Roboto',
+                                                              'fontSize': 55,
+                                                              'fontStyle': 'normal',
+                                                              'fontWeight': 900,
+                                                              'height': 140.0,
+                                                              'left': 0.0,
+                                                              'lineHeight': 'normal',
+                                                              'minWidth': 20,
+                                                              'opacity': 1,
+                                                              'textAlign': 'center',
+                                                              'textDecoration': 'none',
+                                                              'top': 350.0,
+                                                              'width': 1000.0}},
+                                          'resizeVertical': False,
+                                          'type': 'Text'}],
+                            'id': '7vBhS7Y1Z',
+                            'props': {'style': {'backgroundColor': '#F7F7F7'},
+                                      'transition': ['slide']}}],
+                'version': '0.1.3'}}
+
+        self.assertEqual(
+            my_pres['presentation']['version'],
+            exp_pres['presentation']['version']
+        )
+
+        self.assertEqual(
+            my_pres['presentation']['paragraphStyles'],
+            exp_pres['presentation']['paragraphStyles']
+        )
+
+        self.assertEqual(
+            my_pres['presentation']['slidePreviews'],
+            exp_pres['presentation']['slidePreviews']
+        )
+
+        self.assertEqual(
+            len(my_pres['presentation']['slides']),
+            len(exp_pres['presentation']['slides'])
+        )
+
+        self.assertEqual(
+            my_pres['presentation']['slides'][0].keys(),
+            exp_pres['presentation']['slides'][0].keys()
+        )
+
+        self.assertEqual(
+            my_pres['presentation']['slides'][0]['props'],
+            exp_pres['presentation']['slides'][0]['props']
+        )
+
+        for key in my_pres['presentation']['slides'][0]['children'][0].keys():
+            if key not in ['id', 'props']:
+                self.assertEqual(
+                    my_pres['presentation']['slides'][0]['children'][0][key],
+                    exp_pres['presentation']['slides'][0]['children'][0][key]
+                )
+
+        self.assertEqual(
+            my_pres['presentation']['slides'][0]['children'][0]['props'],
+            exp_pres['presentation']['slides'][0]['children'][0]['props']
+        )
```

Comment:
these tests are pretty light- it looks like there is just one test that for a simple presentation, doesn't include `Plotly`, `Image`, multiple slides, or any of the different layout options. Could you try to come up with a few more tests that test these different options?

Reply:
sure
my reason for making it so light was that the JSON blobs are so large anyways
but I agree we should be testing the other features too